app/views/groups/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= stylesheet_link_tag 'groups/index', media: 'all' %>
+<div class="container">
+  <div class="categorie__header">
+    <h1 class="categorie__title"> Categories </h1>
+    <%= link_to destroy_user_session_path do %>
+      <i class="fas fa-sign-out-alt"></i>
+    <% end %>
+  </div>
+
+  <div class="categories__container">
+    <% @categories.each do |categorie| %>
+      <%= link_to group_operations_path(categorie.id) do %>
+        <div class="categorie__item">
+          <img src=<%= categorie.icon %> alt=<%= categorie.name %>>
+          <h2><%= categorie.name %></h2>
+          <p>Total spent:$<span><%= categorie.total_spent %></span></p>
+        </div>
+      <% end %>
+    <% end %>
+  </div>
+  <div class="add__section">
+    <%= link_to new_group_path do %>
+      <button class="categorie__add">Add new Categorie</button>
+    <% end %>
+  </div>
```

Comment:
- [ ] Based on the design guidelines, the **_Add New Category_** button should be at the bottom of the page. I would advise you to implement it this way. It would make your UI better and make your work look identical to that of the template provided. 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/227339954-c274369d-e18f-4df9-86f8-efafe87017f2.png)

### Design Template

![image](https://user-images.githubusercontent.com/77106962/227339445-5fcef1d3-398f-49c3-a8a5-ad8daa1f441c.png)

### Design Requirement
![image](https://user-images.githubusercontent.com/77106962/227340049-c048c907-4775-490d-89dc-dc16ab44609e.png)

Reply:
- [ ] This is well resolved. The button has been positioned properly 👍🏽

<img width="293" alt="Screenshot 2023-03-23 at 21 57 21" src="https://user-images.githubusercontent.com/58692590/227359432-401496a6-d265-4e3b-953e-5c9d16e9170f.png">